--- popa3d-0.5/auth_pam.c~	Mon Oct 22 14:14:00 2001
+++ popa3d-0.5/auth_pam.c	Wed Dec 26 17:11:12 2001
@@ -24,6 +24,11 @@
 #endif
 typedef linux_const void *pam_item_t;
 
+#if AUTH_LIBPAM_USERPASS
+#include <security/pam_userpass.h>
+
+#else
+
 #if AUTH_PAM_USERPASS
 #include <security/pam_client.h>
 
@@ -129,6 +134,7 @@
 
 	return PAM_SUCCESS;
 }
+#endif /* AUTH_LIBPAM_USERPASS */
 
 static int is_user_known(char *user)
 {
