--- popa3d-0.6.3/params.h.orig	2003-04-12 19:44:50 +0400
+++ popa3d-0.6.3/params.h	2003-04-12 21:52:20 +0400
@@ -145,10 +145,10 @@
  * Note that there's no built-in password aging support.
  */
 #define AUTH_PASSWD			0
-#define AUTH_SHADOW			1
+#define AUTH_SHADOW			0
 #define AUTH_PAM			0
-#define AUTH_PAM_USERPASS		0
-#define USE_LIBPAM_USERPASS		0
+#define AUTH_PAM_USERPASS		1
+#define USE_LIBPAM_USERPASS		1
 
 #if AUTH_PAM || AUTH_PAM_USERPASS
 #define AUTH_PAM_SERVICE		POP_SERVER
@@ -163,7 +163,7 @@
  * it harder (but still far from impossible, on most systems) to check
  * for valid usernames. Adjust it for your crypt(3).
  */
-#define AUTH_DUMMY_SALT			"xx"
+#define AUTH_DUMMY_SALT			"$2a$08$xxxxxxxxxxxxxxxxxxxxxx"
 
 #endif
 
